diff --git a/node_modules/@lerna/version/index.js b/node_modules/@lerna/version/index.js
index e4fbf24..6a72fce 100644
--- a/node_modules/@lerna/version/index.js
+++ b/node_modules/@lerna/version/index.js
@@ -631,7 +631,7 @@ class VersionCommand extends Command {
         chain = chain.then(() => {
           this.logger.verbose("version", "Updating root package-lock.json");
           return childProcess
-            .exec("npm", ["install", "--package-lock-only", "--ignore-scripts"], this.execOpts)
+            .exec("npm", ["install", "--package-lock-only", "--ignore-scripts", "--legacy-peer-deps"], this.execOpts)
             .then(() => {
               changedFiles.add(lockfilePath);
             });
